DocSum/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -131,20 +210,28 @@ docker compose up -d
        -F "stream=false"
    ```
 
-Following the validation of all aforementioned microservices, we are now prepared to construct a mega-service.
+> More detailed tests can be found here `cd GenAIExamples/DocSum/test`
```

Comment:
Could you please add the complete usage for megaservice here, as the user will only look into this readme, while UT is designed just for developer, not for users.